app/src/androidTest/java/ch/epfl/sweng/erpa/activities/CreateGameActivityTest.java

```diff
@@ -46,18 +50,26 @@
     public final IntentsTestRule<CreateGameActivity> intentsTestRule = new IntentsTestRule<>(
             CreateGameActivity.class);
     private Resources systemResources;
+    private Game game;
+    private String uuid;
 
     @Before
-    public void setSystemResources() {
+    public void setVariables() {
```

Comment:
Why did you change this?

Reply:
because systemResources is no longer the only variable that is initialized